packages/web-runtime/src/App.vue

```diff
@@ -182,7 +182,7 @@ export default {
     },
 
     sidebarLogoAlt() {
-      return this.$gettext('Navigate to all files page')
+      return this.$gettext('Logo ownCloud, navigate to all files page')
```

Comment:
TODO: use general.name from theme instead of hardcoded `ownCloud`. Needs to use `$gettextInterpolate` then.

Reply:
Addressed in the most recent commit, great idea!